api/utils.go

```diff
@@ -59,3 +60,56 @@ func (api *API) FileSizeCheck(size int64) error {
 	}
 	return nil
 }
+
+func (api *API) getDepositAddress(paymentType string) (string, error) {
+	switch paymentType {
+	case "eth", "rtc":
+		return "0xc7459562777DDf3A1A7afefBE515E8479Bd3FDBD", nil
+	case "btc":
+		return "0", nil
```

Comment:
why not empty string, which is perhaps a bit more of a standard 0-value for strings? or is there another purpose?